README.md

```diff
@@ -1,51 +1,63 @@
+registry.resinstaging.io/resinhup/resinhup-raspberrypi3       1.0  c324b00459f3        2 days ago          241.4 MB
+registry.resinstaging.io/resinhup/resinhup-raspberry-pi2      1.0  4ca1d77c1457        2 days ago          174.3 MB
+registry.resinstaging.io/resinhup/resinhup-raspberry-pi       1.0  4ca1d77c1457        2 days ago          174.3 MB
+registry.resinstaging.io/resinhup/resinhup-intel-nuc          1.0  63cc85875b84        2 days ago          225.3 MB
+registry.resinstaging.io/resinhup/resinhup-beaglebone-black   1.0  7553637ea826        2 days ago          258.3 MB
+
+The images taged as _latest_ are following the HEAD of master branch.
+
+Hint: there is script helper called [docker-build-and-push.sh](https://github.com/resin-os/resinhup/blob/master/scripts/docker-build-and-push.sh) for pushing images to registry. Check its help.
 
 ## How to use
-Prepare the image on the target by running
-```sh
-$ ./prepare.sh
-```
-This will stop all the running containers.
-
-### Run container manually
-Run the container manually using:
-```sh
-[host     ]$ rce run -ti --privileged --rm --net=host --volume /:/host resinhup /bin/bash
-[container]$ python3 /app/resinhup.py --config /app/conf/resinhup.conf --debug
-[container]$ exit
-[host     ]$ reboot
-```
-
-### Run container automatically
-Run the container automatically:
-```sh
-[host] $ rce run -ti --privileged --rm --net=host --volume /:/host resinhup
-[host] $ reboot
-```
+Use run-resinhup.sh [wrapper](https://github.com/resin-os/meta-resin/blob/master/meta-resin-common/recipes-support/resinhup/resinhup/run-resinhup.sh) developed in [meta-resin](https://github.com/resin-os/meta-resin). Check _run-resinhup.sh_ help message for all the configuration you can use. Make sure the _run-resinhup.sh_ along with _update-resin-supervisor_ and _resin-device-progress_ scripts are updated.
+
+Pro Hint: In order to make sure these scripts are updated and run resinhup multiple devices **admins** can use a wrapper on top of _run-resinhup.sh_ called [run-resinhup-ssh.sh](https://github.com/resin-os/meta-resin/blob/master/scripts/resinhup/run-resinhup-ssh.sh). This wrapper is not intended for public use as it requires SSH access to the devices over VPN along with admin permissions for API queries. Check _run-resinhup-ssh.sh_ help message for all the configuration you can use.
+
+## Resinhup architecture
+Currently there are 3 components involved in updating a device:
++ resinhup (docker images)
++ run-resinhup.sh (bash wrapper)
++ run-resinhup-ssh.sh (bash wrapper)
+
+### resinhup
+This component is distributed from this repository as docker images (explained above). The overall workflow of the tool is:
+![Minion](https://github.com/resin-os/resinhup/blob/master/images/resinhup-workflow.png)
```

Comment:
I think you can use a relative path for the image, that way it will also show up in branches